ember.md

```diff
@@ -69,6 +71,42 @@ foo: undefined
 foo: null
 ```
 
+<a name="general-structure--naming-addons"></a>
```

Comment:
These <a> tags aren't needed for linking in Markdown. Check the JavaScript standard doc for examples of how to link without HTML.